__tests__/app.test.js

```diff
@@ -40,4 +40,23 @@ describe('testing for get /api/topics', () => {
             expect(result.body.message).toBe('Path not found')
         })
     })
+})
+
+describe('tests for get /api', () => {
+    test('returns status code 200', () => {
+        return request(app)
+        .get('/api')
+        .expect(200)
+    })
```

Comment:
Since we're checking that we get a 200 back in your next test this test wouldn't really be needed.